src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts

```diff
@@ -76,6 +76,11 @@ export class UBSPersonalInformationComponent extends FormBaseComponent implement
       });
   }
 
+  private _getLastAddresses(addressList: Address[]) {
```

Comment:
there is no need to use _ before the method name
TS private will do the thing

Reply:
Done